### PR TITLE
Add devicename and copy over raw files.

### PR DESCRIPTION
**Pull request by CameronRP**

### CODE REVIEW

File: cptv-convert.py [hunk 1]
```diff
@@ -3,36 +3,47 @@
...
-        mp4_name = reader.timestamp.strftime("%Y%m%d-%H%M%S") + ".mp4"
-        #devicename = reader.devicename
-        devicename = "to-be-added"
-        if devicename == None:
+        fileName = reader.timestamp.strftime("%Y%m%d-%H%M%S")
+        raw_name = fileName + ".cptv"
+        mp4_name = fileName + ".mp4"
+        mp4_temp = join(tempfile.gettempdir(), mp4_name)
+        devicename = reader.device_name
+        if devicename == None or devicename == "":
```
Comment (mjs) [inline cptv-convert.py]: The usual way to handle this is just `if not devicename:`

File: cptv-convert.py [hunk 2]
```diff
@@ -54,6 +65,16 @@ def make_mp4(cptv_file, output_folder, colormap):
...
+    copy_file(mp4_temp, mp4_file)
+    os.remove(mp4_temp)
+    if copy and delete:
+        os.remove(cptv_file)
+
+def copy_file(src, dest):
+    copyfile(src, dest)
+    fs = os.open(dest, os.O_RDONLY)
+    os.fsync(fs)
+    os.close(fs)
```
Comment (mjs) [inline cptv-convert.py:77]: Interesting. I didn't know about this approach. It definitely works? (fine if it does!)

File: trigger-cptv-convert.service [hunk 3]
```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Trigger cptv-convert when USB is plugged in.
+RequiresMountsFor=/media/cp
+DefaultDependencies=no
+
+[Service]
+Type=simple
+Restart=no
+RemainAfterExit=true
+ExecStart=/home/pi/virtualenv/test1/bin/python /home/pi/cptv-convert/cptv-convert.py /var/spool/collected-cptv/ /media/cp/ -c -b
```
Comment (mjs) [inline trigger-cptv-convert.service:10]: These paths aren't going to be right for actual usage but it's fine for now.

**Review by mjs — APPROVED** [review]
Looks good.

File: cptv-convert.py (hunk 2 above)
Comment (CameronRP) [reply cptv-convert.py:77]: Yeah, worked in the testing I did :)

**Pull request merged**